modules/ossm-security-mtls.adoc

```diff
@@ -99,3 +118,19 @@ The default is `TLS_AUTO` and does not specify a version of TLS.
 |`TLSv1_3`
 |TLS version 1.3
 |===
+
+[id="ossm-security-enabling-controlplane_{context}"]
+== Enable strict mTLS in for Mixer telemetry or policy components
```

Comment:
Enabling

And same thing about that these should be separate modules following the proper format.